storio-sqlite/src/androidTest/java/com/pushtorefresh/storio/sqlite/impl/ObservableStreamTest.java

```diff
@@ -98,43 +87,35 @@ public void updateEmission() {
                 .prepare()
                 .executeAsBlocking();
 
-        // Should receive updated users
-        emissionChecker.assertThatNextExpectedValueReceived();
-
-        emissionChecker.assertThatNoExpectedValuesLeft();
-
-        subscription.unsubscribe();
+        emissionChecker.waitAllAndUnsubscribe();
     }
 
     @Test
     public void deleteEmission() {
-        final List<User> usersThatShouldBeSaved = TestFactory.newUsers(10);
-        final List<User> usersThatShouldBeDeleted = TestFactory.newUsers(10);
-        final List<User> allUsers = new ArrayList<User>();
-
-        allUsers.addAll(usersThatShouldBeSaved);
-        allUsers.addAll(usersThatShouldBeDeleted);
-
-        putUsersBlocking(allUsers);
+        final List<User> allUsers = getAllUsersBlocking();
+        final List<User> usersThatShouldBeSaved = new ArrayList<User>();
+        final List<User> usersThatShouldBeDeleted = new ArrayList<User>();
+
+        int pos = 0;
+        for (User user : allUsers) {
+            // will delete last part
+            final boolean save = pos < 5;
+            pos++;
+            if (save) {
+                usersThatShouldBeSaved.add(user);
+            } else {
+                usersThatShouldBeDeleted.add(user);
+            }
+        }
 
         final Queue<List<User>> expectedUsers = new LinkedList<List<User>>();
-
-        expectedUsers.add(allUsers);
         expectedUsers.add(usersThatShouldBeSaved);
```

Comment:
Now I have an issue here. As I can see there is different behavior with `updateEmission`.
Here I haven't initial emission, that I have in `updateEmission` test